crates/codegen/runtime/cargo/src/runtime/parser/lexer/mod.rs

```diff
@@ -101,7 +101,7 @@ pub(crate) trait Lexer {
         ParserResult::r#match(
             vec![Edge::anonymous(Node::terminal(
                 kind,
-                input.content(start.utf8..end.utf8),
+                input.content(start..end),
```

Comment:
I suggest adding a `patch` changeset for this, as it has significant user impact. For example:

> Significant speed up to parser performance by memoizing common operations such as scanning trivia and calculating line/col information.